agent/containers/images/image-test-script.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
+
+# A demonstration of how to use pbench-tool-meister-start with an existing
+# Redis server, and one or more remote
+export CONTROLLER=$(hostname -f)
+
+DISTRO_DEFAULT="centos-8"
+TAG_DEFAULT="a933ae45e"
+REDIS_HOST_DEFAULT=$CONTROLLER
+REDIS_PORT_DEFAULT=17001
+
+while :; do
+    read -p "Specify distribution to test (or press 'ENTER' for default ${DISTRO_DEFAULT}): "
+    DISTRO=${REPLY:-${DISTRO_DEFAULT}}
+    read -p "Specify image tag to test (or press 'ENTER' for default ${TAG_DEFAULT}): "
+    TAG=${REPLY:-${TAG_DEFAULT}}
+
+    printf -- "Pulling tool-data-sink image to test...\n"
+    if podman pull quay.io/pbench/pbench-agent-tool-data-sink-$DISTRO:$TAG
+    then
```

Comment:
To be consistent with `while`/`do`, you would move the `then` to the end of the `if` line, separated by a semi-colon (';').